queries/bash/injections.scm

```diff
@@ -1,11 +1,13 @@
 ((comment) @injection.content 
  (#set! injection.language "comment"))
 
-((regex) @injection.content 
+((regex) @injection.content
+ (#set! injection.language "regex"))
 
-((redirected_statement
-   (heredoc_redirect
-     (heredoc_start) @injection.language))
- (heredoc_body) @injection.content
- (#offset! @injection.content 0 0 -1 0)
- (#downcase! @injection.language))(#set! injection.language "regex"))
+((heredoc_redirect
+  [
+    (heredoc_body)
+    (simple_heredoc_body)
+  ] @injection.content
+  (heredoc_end) @injection.language)
+ (#downcase! @injection.language))
```

Comment:
do we want to support injecting when ${THIS_SH} is detected?

https://git.savannah.gnu.org/tree/tests/arith-for.tests?id=ec8113b9861375e4e17b3307372569d429dec814#n91

Reply:
That link isn't working.

Reply:
updated

Reply:
There are too many cases to cover.

```bash
${THIS_SH} -c '...'
sh -c '...'; /bin/sh -c '...'; /usr/bin/sh -c '...'
bash -c '...'; /bin/bash -c '...'; /usr/bin/bash -c '...'
command sh -c '...' # plus path variants
command bash -c '...' # plus path variants
command -p sh -c '...' # plus path variants
command -p bash -c '...' # plus path variants
env sh -c '...' # plus path variants, plus env options
env bash -c '...' # plus path variants, plus env options
```

Reply:
hm yeah, nevermind